ObsCoreExtensionForRadioData.tex

```diff
@@ -470,20 +466,20 @@ \subsection{Observational configuration and instrumental parameters}
 %	\item Wobble : observations measuring both the source and the surroundings	
 \end{itemize}
 
-\subsection{Auxiliary datasets useful for data quality estimation}
+\subsection{Auxiliary datasets  for data quality estimation}
 
 Auxiliary datasets such as  \emph{uv} distribution map, dirty beam maps, frequency/amplitude plots, phase/amplitude plots are useful for astronomers to check data quality.
 
-In that case DataLink \citep{2023ivoa.spec.1215B} may provide a solution to attach these auxiliary data to ObsCore records. The \texttt{semantics} FIELD in the \{link\}
-response  will contain \#auxiliary  for links to these maps or plots while  the \texttt{content\_qualifier} FIELD introduced from 1.1  could contain a term from a defined vocabulary (to be defined) following the IVOA vocabulary definition  \citep{2021ivoa.spec.0525D}.
+In that case,  DataLink \citep{2023ivoa.spec.1215B} may provide a solution to attach these auxiliary data files to ObsCore records. The link , described in  the specification as  \{link\}
```

Comment:
La link response c'est la table complete. moi j'enleverai "described in the specification as \link response.

Reply:
I think it would be clearer if we show an example of the link response with all fields set up . 
I did not find any example in the JIVE implementation . may be we can mimic a datalink scenario.

Reply:
ok